core/lib/test/unit/async-data-loader.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { AsyncDataLoader, AsyncDataLoaderAction } from '../../async-data-loader';
+
+jest.mock('@vue-storefront/core/lib/logger', () => ({
+  Logger: {
+    info: jest.fn(() => () => {})
+  }
+}))
+
+describe('AsyncDataLoader', () => {
+  it('contains new action with custom category parameter', () => {
+    const actionMock = {
+      execute: jest.fn(),
+      category: 'action1'
+    }
+
+    AsyncDataLoader.push(actionMock);
+
+    const expectedMock = {
+      execute: actionMock.execute,
+      category: 'action1',
+      scheduledAt: (actionMock as AsyncDataLoaderAction).scheduledAt
+    }
+
+    expect(AsyncDataLoader.queue[0].scheduledAt).toBe(expectedMock.scheduledAt)
+    expect(AsyncDataLoader.queue[0].category).toBe(expectedMock.category)
+    expect(AsyncDataLoader.queue[0].execute).toBe(expectedMock.execute)
+  })
+
+  it('contains new action with default category parameter', () => {
+    const actionMock = {
+      execute: jest.fn()
+    }
+
+    AsyncDataLoader.push(actionMock);
+
+    const expectedMock = {
+      execute: actionMock.execute,
+      category: 'asyncData',
```

Comment:
Changing value of `DEFAULT_ACTION_CATEGORY` would break this test or give false results. `export` this const, import it there and use it